builtin/game/chatcommands.lua

```diff
@@ -679,19 +679,35 @@ core.register_chatcommand("status", {
 })
 
 core.register_chatcommand("time", {
-	params = "<0...24000>",
+	params = "<0..23>:<0..59>",
 	description = "set time of day",
 	privs = {settime=true},
 	func = function(name, param)
 		if param == "" then
-			return false, "Missing time."
+			local current_time  = math.floor(core.get_timeofday() * 1440)
+			return true, ("Current time is %2d:%2d"):format((current_time - (current_time % 60)) / 60,(current_time % 60))
```

Comment:
this line is too long, can you split it up? also you miss a space between `,` and `(`